### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by prateekcr**
Updating README to reflect the changes made in this release.

Signed-off-by: Chandrakar, Prateek <prateek.chandrakar@intel.com>

### CODE REVIEW

**Review by trbehera — APPROVED** [review]

**Review by trbehera — CHANGES_REQUESTED** [review]
Needs some more change.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Secure Device Onboard (SDO) Client SDK
+# FIDO Device Onboard (FDO) Client SDK
 
 [ Introduction ](docs/introduction.md) <br>
-[  SDO Compilation Setup ](docs/setup.md)
+[  FDO Compilation Setup ](docs/setup.md)
 - [ Safestring ](docs/setup.md#safestring) <br>
-- [ Service Info Modules ](docs/setup.md#si_info_modules)
+- [ TinyCBOR ](docs/setup.md#tinycbor) <br>
```
[the remark this answers is not in the document]
Comment (prateekcr) [reply README.md:6]: There was no such section in setup.md. This was a broken link that has been removed.

**Comment by prateekcr** [comment]
All comments, except 1, have been updated in the commit.

**Review by trbehera — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]
take car3 of tushar's comments and otherwise good to go.

**Pull request merged**